src/core/mount.c

```diff
@@ -1020,7 +1020,9 @@ static void mount_enter_mounting(Mount *m) {
         if (r < 0)
                 goto fail;
 
-        (void) mkdir_p_label(m->where, m->directory_mode);
+        r = mkdir_p_label(m->where, m->directory_mode);
+        if (r < 0 && r != -EEXIST)
+                log_unit_warning_errno(UNIT(m), r, "Failed to create mount point '%s': %m", m->where);
```

Comment:
"...%m, ignoring."

Also, it should be a debug, or at most info I think

Reply:
"Failed to create mount point '%s', ignoring: %m"

Reply:
Since the warning is done in a more generic function now, we can't assume it's for a mount point so I kept the log message generic as well.

Added "ignoring".